generate_golden.sh

```diff
@@ -28,63 +28,86 @@ build_kb() {
 
-		$kb init --project-version $version --domain testproject.org --license apache2 --owner "The Kubernetes authors"
-		$kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
-		$kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
-		$kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
-		$kb create webhook --group crew --version v1 --kind FirstMate --conversion
-		$kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
-		# TODO(droot): Adding a second group is a valid test case and kubebuilder is expected to report an error in this case. It
-		# doesn't do that currently so leaving it commented so that we can enable it later.
-		# $kb create api --group ship --version v1beta1 --kind Frigate --example=false --controller=true --resource=true --make=false
-		$kb create api --group core --version v1 --kind Namespace --example=false --controller=true --resource=false --namespaced=false --make=false
-		# $kb create api --group policy --version v1beta1 --kind HealthCheckPolicy --example=false --controller=true --resource=true --namespaced=false --make=false
-	fi
-	make all test # v2 doesn't test by default
-	rm -f Gopkg.lock
-	rm -f go.sum
-	rm -rf ./vendor
-	rm -rf ./bin
-	export GOPATH=$oldgopath
-	popd
+            header_text 'initializing  project-v2-multigroup ...'
+            $kb init --project-version $version --domain testproject.org --license apache2 --owner "The Kubernetes authors"
+
+            header_text 'Switching to multigroup layout ...'
+            $kb edit --multigroup=true
+
+            header_text 'Creating APIs ...'
+            $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
+            $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
+            $kb create api --group ship --version v1beta1 --kind Sailor --controller=true --resource=true --make=false
+            $kb create webhook --group ship --version v1beta1 --kind Sailor --conversion
+            $kb create api --group ship --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
+            $kb create api --group ship --version v2alpha1 --kind Cruiser --controller=true --resource=true --namespaced=false --make=false
+            $kb create api --group sea-creatures --version v1beta1 --kind Kraken --controller=true --resource=true --make=false
+            $kb create api --group sea-creatures --version v1beta2 --kind Leviathan --controller=true --resource=true --make=false
+            $kb create api --group core.policy --version v1 --kind HealthCheckPolicy --controller=true --resource=true --make=false
```

Comment:
> core.policy

Using `core` here can be potential misleading.
Just call `policy`, or just drop this line is also fine.

Reply:
If you want to exercise format like `foo.bar`here.
You can probably use `thirdparty.policy` or `foo.policy`.